components/openvino/model_convert/README.md

```diff
@@ -0,0 +1,268 @@
+# Model optimization component
```

Comment:
Would it be possible to move the code to /contrib/components/openvino?

Reply:
sure, moved according to this suggestion